Sources/Rownd/framework/Customizations.swift

```diff
@@ -41,7 +41,7 @@ open class RowndCustomizations: Encodable {
         )
 
         aniView.loopMode = .loop
-        aniView.startAnimating()
+        aniView.play()
```

Comment:
Does this change the behavior vs. the `startAnimating` extension we built?

Reply:
Not that I know of. I tested it and it seems to be behaving normally